docs/fit/index.rst

```diff
@@ -263,13 +293,17 @@ The :py:class:`~sherpa.plot.DataPlot`, :py:class:`~sherpa.plot.ModelPlot`,
 and :py:class:`~sherpa.plot.FitPlot` classes can be used to
 :doc:`see how well the model represents the data <../plots/index>`.
 
-   >>> from sherpa.plot import DataPlot, ModelPlot
-   >>> dplot = DataPlot()
-   >>> dplot.prepare(f.data)
-   >>> mplot = ModelPlot()
-   >>> mplot.prepare(f.data, f.model)
-   >>> dplot.plot()
-   >>> mplot.overplot()
+.. plot::
+    :include-source:
+    :context:
+
+    >>> from sherpa.plot import DataPlot, ModelPlot
+    >>> dplot = DataPlot()
+    >>> dplot.prepare(f.data)
+    >>> mplot = ModelPlot()
+    >>> mplot.prepare(f.data, f.model)
+    >>> dplot.plot()
+    >>> mplot.overplot()
 
 .. image:: ../_static/fit/data_model_c0_c2.png
```

Comment:
Can we not delete this figure too? 

As mentioned above, I have paged out all the plot/doc knowledge I used to have...

Reply:
Yes we can. Not why I didn't (of, if I did, why it doesn't show up here. GH tell me the lines you commend on are "outdated" some maybe I did in a later commit. Either way, will check.)

Reply:
[do not understand why this is labelled outdated; not sure I want to know, as I don't think I was looking at an old version, but maybe I was ???]

Reply:
It can happen when you comment "per commit" and the code was changed in a later commit then the one you looked at while you wrote that comment.